### PR TITLE
docs: changed bugs and feature link

### PR DESCRIPTION
**Pull request by rjoydip**
Changed bug and feature links in readme and some styling issue (`:`). 

This will filter out issues based on label which tagged with issue.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,9 +19,9 @@
   <p align="center">
     <a href="#demo">View Demo</a>
     ·
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Report Bug</a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aissue+is%3Aopen+label%3Abug">Report Bug</a>
     ·
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Request Feature</a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aissue+is%3Aopen+label%3Afeature">Request Feature</a>
```
Comment (anuraghazra) [inline readme.md:24]: Hi @rjoydip thanks for the PR, i just realized it's would be better if the URL is this https://github.com/anuraghazra/github-readme-stats/issues/new/choose

Comment (rjoydip) [reply readme.md]: Done

**Comment by izzyhudson43** [comment]
True

**Comment by anuraghazra** [comment]
Hey @rjoydip sorry for the late response, thanks for the PR

**Pull request merged**